source/adios2/toolkit/format/bp5/BP5Deserializer.h

```diff
@@ -174,6 +174,7 @@ class BP5Deserializer : virtual public BP5Base
     std::vector<void *> *m_FreeableMBA = nullptr;
 
     std::vector<void *> *m_JoinedDimenOffsetArrays = nullptr;
+    std::vector<void *> *m_FreeableJDOA = nullptr;
```

Comment:
This attribute needs to the last one in order not to break the ABI compatibility

Reply:
> would it make sense to put this version in master and merge a version to release_29 with it relocated to the end?

Yes, that is a good idea.

Reply:
Actually, I think this isn't an ABI issue.  BP5Deserializer is a completely ADIOS-internal class.  You'd only get an ABI issue if this was an externally-facing class and we were changing its public members, right?  Here, if the user ends up pointing to a new version of the ADIOS library he'll end up new code that references this class as well as the new class itself.  Or is there something wrong with my reasoning?